src/components/dashboard/quick-action.tsx

```
@@ -21,15 +22,26 @@ const QuickAction: React.FC<QuickActionProps> = (props) => {
-          </Button>
-          <Button variant="outline" className="w-full">
-            <Users className="mr-2 h-4 w-4" /> Invite Team Member
-          </Button>
+          <Protect
+            permission="org:sys_members:manage"
+            fallback={
+              <>
+                <p className="text-xl uppercase text-center text-secondary-foreground">
+                  Your role does not grant you an access to this
+                </p>
+              </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)